### PR TITLE
Add compilers used in Umpire

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

File: spack_configs/blueos_3_ppc64le_ib/packages.yaml [hunk 1]
```diff
@@ -12,9 +12,9 @@ packages:
     buildable: False
 
   cuda:
-    version: [10.1.243,10.1.168,9.2.148,8.0]
+    version: [10.1.168,11.0.2,9.2.148,8.0]
     paths:
-      cuda@10.1.243: /usr/tce/packages/cuda/cuda-10.1.243
+      cuda@11.0.2: /usr/tce/packages/cuda/cuda-11.0.2
```
Comment (adrienbernede) [inline spack_configs/blueos_3_ppc64le_ib/packages.yaml:17]: 10.1.243 is what RAJA is using, 10.1.168 is what Umpire is using.
Can you confirm that you want RAJA to move to 10.1.168?

File: spack_configs/blueos_3_ppc64le_ib/compilers.yaml [hunk 2]
```diff
@@ -26,10 +26,10 @@ compilers:
     environment: {}
     extra_rpaths: []
 - compiler:
-    spec: clang@8.0.1
+    spec: clang@9.0.0
```
Comment (adrienbernede) [inline spack_configs/blueos_3_ppc64le_ib/compilers.yaml:29]: RAJA is using chang@8.0.1 in its scripts, can you confirm they won’t need it?

**Review by adrienbernede — COMMENTED** [review]
This looks good!
I just have a question about compatibility with RAJA (cuda 10).
You also removed clang8.0.1 which RAJA uses in its test scripts.
Please confirm ;)

File: spack_configs/blueos_3_ppc64le_ib/packages.yaml (hunk 1 above)
Comment (davidbeckingsale) [reply spack_configs/blueos_3_ppc64le_ib/packages.yaml:17]: I'll keep both.

File: spack_configs/blueos_3_ppc64le_ib/compilers.yaml (hunk 2 above)
Comment (davidbeckingsale) [reply spack_configs/blueos_3_ppc64le_ib/compilers.yaml:29]: They will be using the same configs as Umpire 😄 but I think we should keep 8.0.1 too.

Comment (adrienbernede) [reply spack_configs/blueos_3_ppc64le_ib/compilers.yaml:29]: You tell me. At this point we don’t have to keep more config than we need.

Comment (davidbeckingsale) [reply spack_configs/blueos_3_ppc64le_ib/compilers.yaml:29]: I've added CUDA 10.1.243, but won't do clang 8. I want the RAJA jobs to match the Umpire ones.

**Comment by adrienbernede** [comment]
I have PRs opened in RAJA, let me know if I should update them with this, or wait for them to be merged and them submit an update.

**Comment by davidbeckingsale** [comment]
> @davidbeckingsale I have PRs opened in RAJA, let me know if I should update them with this, or wait for them to be merged and them submit an update.

Please update them.

**Pull request merged**